terms/experimentalData/differentiationMethod.json

```diff
@@ -0,0 +1,17 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "sage.annotations-experimentalData.differentiationMethod-0.0.1",
+  "description": "The general strategy by which a reprogrammed or pluripotent cell is differentiated into a target cell.",
+  "anyOf": [
+        {
+            "const": "directed differentiation",
+            "description": "Cell differentiation via the addition of growth factors and small molecules that modulate developmental signaling pathways.",
+            "source": "https://doi.org/10.1016/j.ymeth.2015.11.019"
```

Comment:
Should this source be the same as the Ngn2 one?

Reply:
Yeah, it's a review paper that explains both terms.